scripts/common.py

```diff
@@ -254,7 +253,7 @@ def on_linux() -> bool:
     return platform.system() == "Linux"
 
 
-def regex(raw: str):
+def regex(raw: str) -> 're.Pattern':
```

Comment:
Why does it need to be quoted?  Does that have to do with that Ubuntu 18 issue?

Reply:
Yes, Ubuntu 18's python is too old to know the `re.Pattern` type, so this is how it must be annotated.